python/tvm/contrib/graph_executor.py

```diff
@@ -195,6 +198,49 @@ def set_input(self, key=None, value=None, **params):
                 if val:
                     self._get_input(k).copyfrom(params[k])
 
+    def set_output(self, key, value):
+        """Set outputs to the module
+
+        Parameters
+        ----------
+        key : int or str
+           The output key
+
+        value : the output value
+           The output value
+        """
+        self._set_output(key, value)
+
+    def set_input_zero_copy(self, key, value, **params):
+        """Set inputs to the module via kwargs with zero memory copy
+
+        Parameters
+        ----------
+        key : int or str
+           The input key
+
+        value : the input value in DLPack
+           The input key
+
+        params : dict of str to NDArray
+           Additional arguments
+        """
+        self._set_input_zero_copy(key, value)
+        self.set_input(None, None, **params)
```

Comment:
not needed

Reply:
I was hoping that we can keep the apis consistent with `set_input` so that is the reason I kept `**params`. Should we still remove it?

Reply:
We can support `params`. However, we can't call `set_input` as it involves with copying.

Reply:
good point. Updated this method to use zero copy.